sdf/mandelbulbSDF.glsl

```diff
@@ -0,0 +1,39 @@
+#include "../space/spherical.glsl"
+
+/*
+author: Kathy McGuiness
```

Comment:
this is headers are structured in YAML format to compose the documentation in lygia.xyz/sdf. the field for author is: `original author`